src/main/resources/spring/camel-context.xml

```diff
@@ -38,6 +38,20 @@
     <bean id="utils" class="org.jboss.xavier.utils.Utils"/>
 
     <camelContext id="camel" xmlns="http://camel.apache.org/schema/spring">
+        <onException>
+            <exception>java.util.NoSuchElementException</exception>
+            <handled><constant>true</constant></handled>
+            <setHeader headerName="exception">
+                <simple>${exception.stacktrace}</simple>
+            </setHeader>
+            <setHeader headerName="Exchange.HTTP_RESPONSE_CODE">
+                <simple resultType="String">${type:javax.servlet.http.HttpServletResponse.SC_NOT_FOUND}</simple>
+            </setHeader>
+            <setBody>
+                <simple>${exception.message}</simple>
+            </setBody>
+            <stop/>
+        </onException>
```

Comment:
I think this block of code is being duplicated. You can see that `<exception>java.util.NoSuchElementException</exception>` is defined twice in https://github.com/project-xavier/xavier-integration/blob/44d270fbd51aa70496b74bf10e646bbafeeb944a/src/main/resources/spring/camel-context.xml#L42

Reply:
+1 totally..... weird .... I assume my inexperience with vs code and merging conflicts ;)